sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusNonFederatedIntegrationTest.java

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.servicebus;
+
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
+import com.azure.core.credential.AzureNamedKeyCredential;
+import com.azure.core.credential.AzureSasCredential;
+import com.azure.core.test.TestMode;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.administration.ServiceBusAdministrationAsyncClient;
+import com.azure.messaging.servicebus.administration.ServiceBusAdministrationClientBuilder;
+import com.azure.messaging.servicebus.administration.models.NamespaceProperties;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.opentest4j.TestAbortedException;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.azure.messaging.servicebus.TestUtils.getConnectionString;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+/**
+ * Live tests with non-federated authentication.
+ */
+@Tag("integration")
+@Execution(ExecutionMode.SAME_THREAD)
+public class ServiceBusNonFederatedIntegrationTest extends IntegrationTestBase {
```

Comment:
There were 3 tests that use Sas (derived from connection string), they are now moved to this dedicated test class. If the connection string environment var is set, then these tests will be run. Ideally these 3 could run in auxiliary tenant once available.

Reply:
While these 3 tests seem eligible for aux tenant, at the moment they will currently run in CI (ARM template output connection string). All other tests use Federated Managed Identity.